src/server.py

```diff
@@ -411,6 +428,29 @@ def _ensure_release_countdown(self):
         self._countdown_id = GLib.timeout_add_seconds(
             self._TIMEOUT_S, self.release, priority=GLib.PRIORITY_LOW)
 
+    def _priority_playing(self):
+        for uuid_ in self.players:
+            if self.players[uuid_].priority:
+                return True
+        return False
+
+    def _apply_on_loops(self, uuid, method):
```

Comment:
`uuid` could be something more meaningful like `prioritary_uuid` or `current_uuid`

Reply:
That is just a regular event uuid, no different from other uuid's used in the server.py, I think I should keep it consistent with the rest of the code.

Reply:
or `uuid_to_skip`